### DISCUSSION TITLE
More granular network permission in edge-runtime

### DISCUSSION THREAD

**Discussion by 0x5457** [supabase/supabase#27341] [Feature Requests] [answered]
When creating userWorkers in edge-runtime, it's possible to disable network access using the `netAccessDisabled` option. However, there are situations where I would like to restrict userWorkers to only access localhost. I am wondering if edge-runtime could support a more refined network policy, such as `allowNet: [ip:port]`. Your guidance on this matter would be greatly appreciated.

**Comment by 0x5457** [comment, marked as answer]
I tried to submit a [PR](https://github.com/supabase/edge-runtime/pull/386) to support this

**Comment by Codexiaoyi** [comment]
This feature is very much needed!!!
